app/src/androidTest/java/com/github/h3lp3rs/h3lp/forum/ForumAnswersActivityTest.kt

```diff
@@ -62,8 +66,20 @@ class ForumAnswersActivityTest {
             ), emptyList()
         )
 
-        ActivityScenario.launch<ForumAnswersActivity>(intent)
+        globalContext = getApplicationContext()
+        userUid = USER_ID_NAME
```

Comment:
Just for consistency, you can use the already defined USER_TEST_ID for the UID since it's different from the user name